src/main/java/uk/gov/companieshouse/insolvency/delta/processor/InsolvencyDeltaProcessor.java

```diff
@@ -55,11 +53,35 @@ public void processDelta(Message<ChsDelta> chsDelta,
         final ChsDelta payload = chsDelta.getPayload();
         final String logContext = payload.getContextId();
         final Map<String, Object> logMap = new HashMap<>();
+        final String companyNumber;
 
-        InsolvencyDelta insolvencyDelta = mapToInsolvencyDelta(payload);
+        if (Boolean.TRUE.equals(payload.getIsDelete())) {
```

Comment:
This decision should be made in InsolvencyDeltaConsumer for better separation with separate processors for delete and deltas. Ex: if(delete) {
deleteProcessor.processDelete
} else {
deltaProcessor.processDelta
}

Reply:
Okay happy to accommodate

Reply:
Done